prime-router/docs/playbooks/add_ip_deny_list.md

```diff
@@ -22,9 +22,10 @@ There may be a time where we need to blacklist a specific IP address due to a ba
     - Then
       - Deny Traffic
 
-### Amazon Cli
+### Azure CLI
 1. Add a custom WAF Rule with the following changes: 
-- **IPAllowPolicyExampleCLI**: The Policy name for the WAF Policy
+- **IPAllowPolicyExampleCLI**: The Policy name for the WAF Policy you wish to create.
+- **IPAllowListRule**: Can be used multiple times, but should be a description of the specific rule you are creating.
 - Priority: Based on other rules
 - resource-group: The Resource Group you are adding the rule in
 - ip-address-range-1: IP address you want to block
```

Comment:
Is there reference material that specifies the format of ip-address-range?  The console example implies that CIDR notation is allowed.  Can we also specify in other formats like "1.2.3.3-7"?